intellij-ext/src/main/kotlin/com/metalbear/mirrord/MirrordListener.kt

```diff
@@ -102,8 +130,21 @@ class MirrordListener : ExecutionListener {
         super.processTerminating(executorId, env, handler)
     }
 
-    private fun getRunConfigEnv(env: ExecutionEnvironment): LinkedHashMap<String, String> {
-        val envMethod = env.runProfile.javaClass.getMethod("getEnvs")
-        return envMethod.invoke(env.runProfile) as LinkedHashMap<String, String>
+    private fun getRunConfigEnv(env: ExecutionEnvironment): LinkedHashMap<String, String>? {
+        return if (env.runProfile::class.simpleName == "GoApplicationConfiguration")
```

Comment:
```suggestion
        if (env.runProfile::class.simpleName == "GoApplicationConfiguration") return null
```
Then the rest of the function, without the else. Lets you avoid needless nesting, which makes functions less readable.